source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -282,8 +282,9 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
     envoy::type::FractionalPercent random_sampling;
     // TODO: Random sampling historically was an integer and default to out of 10,000. We should
```

Comment:
Is this TODO still relevant? I miss context here. @mattklein123 @objectiser ?

Reply:
I think it is still relevant, as `random_sampling` still waits to be migrated to fractional_percent.